api/src/main/scala/pl/touk/nussknacker/engine/api/exception/NuExceptionInfo.scala

```diff
@@ -2,4 +2,4 @@ package pl.touk.nussknacker.engine.api.exception
 
 import pl.touk.nussknacker.engine.api.Context
 
-case class NuExceptionInfo[T <: Throwable](nodeId: Option[String], throwable: T, context: Context) extends Serializable
\ No newline at end of file
+case class NuExceptionInfo[T <: Throwable](nodeId: Option[String], componentName: Option[String], componentType: Option[String], throwable: T, context: Context) extends Serializable
```

Comment:
Please use `ComponentType` for type

Reply:
1. Ok
2. For basic components (like Filter or Switch) the `componentName` was actually empty, but `componentType` was `Some("filter")`. Maybe for basic component we can use the same value for `componentName` and `componentType`, WDYT?

Reply:
2. IMO same value is ok. In the far future I would like to replace base components with components with type = custom and name = e.g. filter so this option is close to this vision.

Reply:
Done :)